.vscode/cspell.json

```
@@ -14,6 +14,8 @@
     "hugo",
     "initializers",
     "Istio",
+    "Jaeger",
```

I suggest lowercase here and also adding a textlint rule for the proper capitalization. WDYT?

```suggestion
    "jaeger",
```